homework/src/Repository.js

```diff
@@ -16,19 +16,50 @@ class Repository {
     //
     // Replace this comment with your code
     //
+
+    const table = Util.createAndAppend('table', parent);
+    const tr = Util.createAndAppend('tr', table);
+    Util.createAndAppend('td', tr, { 'html': 'Repository  : ', 'class': 'tr' });
+    Util.createAndAppend('td', tr, { 'html': this.repo.name });
```

Comment:
You have repeated code here: you are creating a `tr` with two `td` children three times. You could pply the DRY principle by creating a method, e.g. `addRow`:

```js

render(parent) {
  const table = Util.createAndAppend('table', parent);
  this.addRow(table, 'Repository :', this.repo.name);
  //...
}

addRow(table, label, value) {
  //...
}
```